ONSdigital/sdx-transform-cs
# Add comment being passed through to survey

**Changes**

Add comment being passed from eq to survey data. Fixes #20 

**How to test**

Add a comment to eq payload from console (question number 146), see if it appears in the image produced.

**Who can test**

Anyone but @iwootten
